.github/workflows/checks.yml

```
@@ -9,7 +9,7 @@ jobs:
     strategy:
       matrix:
         # For quotes on 3.10: https://github.com/actions/setup-python/issues/160
-        python_version: [3.8, 3.9, '3.10']
+        python_version: [3.8, 3.9, '3.10', 3.11]
```

```suggestion
        python_version: [3.8, 3.9, '3.10', '3.11']
```
If I remember the root cause correctly, we should probably add these to 3.11 too.

---

Reason for not putting them (in Python, think it is universal):

```python
>>> float("3.10")
3.1
>>> float("3.11")
3.11
```

if not `.0` then quotes are not needed. Left it for Python 3.20 :smile: